tests/gtest/avifreadimagetest.cc

```diff
@@ -370,6 +370,88 @@ TEST(ICCTest, RGB2Gray2RGB) {
   }
 }
 
+TEST(GrayTest, Roundtrip) {
+  constexpr char file_name[] = "paris_icc_exif_xmp.png";
+  const std::string file_path = std::string(data_path) + file_name;
+
+  // Read the ground truth image as gray.
+  ImagePtr image(avifImageCreateEmpty());
+  ASSERT_NE(image, nullptr);
+  ASSERT_NE(avifReadImageForRGB2Gray2RGB(file_path, AVIF_PIXEL_FORMAT_YUV400,
+                                         /*ignore_icc=*/true, image),
+            AVIF_APP_FILE_FORMAT_UNKNOWN);
+  for (const std::string ext : {"png", "jpg"}) {
```

Comment:
These could be in alphabetical order: jpg, png.

Reply:
Done

Reply:
Looks like this function missed an update, the previous one got one.

Reply:
I see `for (const std::string ext : {"png", "jpg"}) {` here, but `for (const std::string ext : {"jpg", "png"}) {` at line 336 now.

Reply:
Done